src/scrabble.js

```diff
@@ -1,6 +1,63 @@
 class Scrabble {
   // Write your implementation here
 
+  constructor (wordValue) {
+    this.wordValue = wordValue
+    this.isDouble = false
+    this.isTriple = false
+    this.obj = {
+      value1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+      value2: ['D', 'G'],
+      value3: ['B', 'C', 'M', 'P'],
+      value4: ['F', 'H', 'V', 'W', 'Y'],
+      value5: ['K'],
+      value8: ['J', 'X'],
+      value10: ['Q', 'Z']
+    }
+  }
+
+  _validateString (word) {
+    if (typeof word !== 'string' || word.length === 0) {
+      return false
+    } else return true
+  }
```

Comment:
Is there a need to pass the `word` param into the function ? Since you're inside an object (`Scrabble`) you could have empty params and use `this.wordValue` instead of `word`.

Or maybe there's a reason for using params ?

Reply:
Nice use of arrays

Reply:
meat to validate any string later come up as if the application could grow